.github/workflows/release-debug-base.yaml

```diff
@@ -0,0 +1,38 @@
+name: Create Debug Base Image Release
+
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      - apko.yaml
+
+jobs:
+  build:
+    name: Release Debug Base OCI image
+    runs-on: ubuntu-latest
+
+    # https://docs.github.com/en/actions/reference/authentication-in-a-workflow
+    permissions:
+      id-token: write
+      packages: write
+      contents: read
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - id: apko
+        uses: chainguard-images/actions/apko-snapshot@main
+        with:
+          config: apko.yaml
+          image_refs: apko.images
+          base-tag: ghcr.io/${{ github.repository }}/debug-base
+          target-tag: latest
+
+      - name: Emit Image Refs output
```

Comment:
I think we could just skip this step. If I need to see the digests of the base image I can just use crane.